tests/conftest.py

```diff
@@ -15,6 +15,32 @@
 
 from . import helpers
 from .helpers.printer import wait_for_output
+import pytest_timeout
+import tempfile
+
+_original_dump_stacks = pytest_timeout.dump_stacks
+
+
+def _print_pydevd_log(reason):
+    sys.stderr.write('\n******************************************************************\n')
```

Comment:
Note: with this change, besides the ptvsd log and thread information the pydevd output is also sent (I had to add it to diagnose why some things were not working on the ci).